account_billing/views/account_billing_views.xml

```diff
@@ -0,0 +1,251 @@
+                            <field
+                                name="invoice_related_count"
+                                string="Invoices"
+                                widget="statinfo"
+                            />
+                        </button>
+                    </div>
+                    <h1>
+                        <label
+                            for="name"
+                            string="Draft Billing"
+                            attrs="{'invisible': [('state','!=', 'draft')]}"
+                        />
+                        <field
+                            name="name"
+                            readonly="1"
+                            class="oe_inline"
+                            attrs="{'invisible': [('state', '=', 'draft')]}"
+                        />
+                    </h1>
+                    <group>
+                        <group>
+                            <field
+                                name="partner_id"
+                                attrs="{'readonly': [('state', '!=', 'draft')]}"
+                            />
+                            <field name="bill_type" invisible="1" />
+                        </group>
+                        <group>
+                            <field name="threshold_date_type" />
+                            <field name="threshold_date" string="Threshold Date" />
+                            <field name="date" />
+                            <field name="currency_id" />
+                        </group>
+                    </group>
+                    <notebook>
+                        <page string="Billing Lines" groups="base.group_user">
+                            <field name="billing_line_ids">
+                                <tree name="billing" create="0" editable="bottom">
+                                    <field name="invoice_id" />
+                                    <field name="name" readonly="1" force_save="1" />
+                                    <field
+                                        name="invoice_date"
+                                        readonly="1"
+                                        force_save="1"
+                                    />
+                                    <field
+                                        name="threshold_date"
+                                        readonly="1"
+                                        force_save="1"
+                                    />
+                                    <field name="origin" readonly="1" force_save="1" />
+                                    <field
+                                        name="total"
+                                        sum="Total Amount"
+                                        readonly="1"
+                                        force_save="1"
+                                    />
+                                    <field name="state" readonly="1" force_save="1" />
```

Comment:
Would be nice to use badges here like base Odoo?
                <field name="state" widget="badge" decoration-success="state == 'done'" decoration-info="state in ('draft', 'in_progress')"/>